content/android/getting-started.md

```diff
@@ -119,10 +114,37 @@ Also, you need to make sure your application has grant
 <uses-permission android:name="android.permission.INTERNET" />
 ```
 
-<a name="what's-next"></a>
+<a name="create"></a>
+## Creating a new project with configured SDK
+
+### Step 1: Installing Android Studio
+
+We recommend developing Android apps with Skygear Android SDK using
+[Android Studio](https://developer.android.com/studio/index.html).
+
+### Step 2: Downloading scaffolding project
+
+Clone the repo on github
```

Comment:
I think you can say "Download" instead of "Clone", so that the direct `.zip` link is not necessary